src/main/webapp/js/components/sidebar.jsx

```diff
@@ -266,59 +274,59 @@ class SideBar extends Component {
           <ul className="nav navbar-nav side-nav">
             <li>
               <Link to="/vaccinee">
-                <FontAwesomeIcon icon="syringe" />
+                <FontAwesomeIcon className="menu-icon" icon="syringe" />
```

Comment:
I think instead of adding a new class it will be better to add a margin to the "icon-text"